commerce-subscription-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -164,6 +166,19 @@ boolean hasManageCommerceSubscriptionEntryPermission = commerceSubscriptionEntry
 						value="<%= String.valueOf(commerceOrder.getCommerceAccountId()) %>"
 					/>
 
+					<%
+					String paymentMethod = LanguageUtil.get(request, commerceOrder.getCommercePaymentMethodKey());
+
+					if (!commerceSubscriptionEntryDisplayContext.isPaymentMethodActive(commerceOrder.getCommercePaymentMethodKey())) {
+						paymentMethod += StringPool.DOUBLE_SPACE + StringPool.OPEN_PARENTHESIS + LanguageUtil.get(request, "inactive") + StringPool.CLOSE_PARENTHESIS;
```

Comment:
we should have a language key for this. concatenating stuff to a localized value is always wrong. (or at least almost always, but I can't think of an exception atm.) the key can just be like `x-inactive`, and we can pass the other localized value in.